components/base-war/src/main/webapp/skins/phenotips/header.vm

```diff
@@ -41,6 +41,8 @@
     </a>
   </div> ## companylogo
   </div> ## company
+  #set ($displayLanguageChoice = true)
```

Comment:
Actually I agree with Marta, since the variable is only used once, better remove it.